src/mame/mr/nl_dribling.h

```diff
@@ -0,0 +1,12 @@
+// license:BSD-3-Clause
+// copyright-holders: Enzo Lombardi
+
+#ifndef _NL_DRIBBLING_H
+#define _NL_DRIBBLING_H
+#include "netlist/nl_setup.h"
+
+NETLIST_EXTERNAL(LM339)
+NETLIST_EXTERNAL(calcio)
+NETLIST_EXTERNAL(stop_palla_parata)
+NETLIST_EXTERNAL(dribling)
```

Comment:
You don’t declare things `extern` when they are static.

Reply:
Done